packages/core/src/TimePicker/TimePicker.js

```diff
@@ -0,0 +1,725 @@
+import React, { useState, useEffect, useRef, useMemo } from "react";
+import PropTypes from "prop-types";
+import clsx from "clsx";
+import { withStyles } from "@material-ui/core";
+
+import { Time as TimeIcon } from "@hitachivantara/uikit-react-icons";
+import {
+  HvFormElement,
+  HvBaseDropdown,
+  HvLabel,
+  HvWarningText,
+  HvInfoMessage,
+  setId,
+  useUniqueId,
+  useControlled,
+  useLocale,
+  HvTypography,
+  useSavedState,
+} from "..";
+
+import { TimePickerUnits, TimeFormat, PeriodPickerOptions } from "./enums";
+import { getFormattedTime, getTimeFormatForLocale } from "./timePickerFormatter";
+import { getHoursForTimeFormat, getTimeWithFormat24 } from "./timePickerConverter";
+import UnitTimePicker from "./UnitTimePicker";
+import PeriodPicker from "./PeriodPicker";
+
+import styles from "./styles";
+
+const setFocusToContent = (containerRef) => {
+  containerRef?.getElementsByTagName("input")[0]?.focus();
+};
+
+const timeIsEqual = (timeA, timeB) => {
+  return (
+    timeA === timeB ||
+    (timeA == null && timeB == null) ||
+    (timeA != null &&
+      timeB != null &&
+      timeA.hours === timeB.hours &&
+      timeA.minutes === timeB.minutes &&
+      timeA.seconds === timeB.seconds &&
+      timeA.period === timeB.period)
+  );
+};
+
+const timeIsValid = (time, timeFormat) => {
+  const hourInputState =
+    time?.hours != null &&
+    time.hours !== "" &&
+    time.hours >= 0 &&
+    ((timeFormat === TimeFormat.H24 && time.hours <= 24) ||
+      (timeFormat === TimeFormat.H12 && time.hours <= 12));
+  const minutesInputState =
+    time?.minutes != null && time.minutes !== "" && time.minutes >= 0 && time.minutes <= 59;
+  const secondsInputState =
+    time?.seconds != null && time.seconds !== "" && time.seconds >= 0 && time.seconds <= 59;
+
+  return hourInputState && minutesInputState && secondsInputState;
+};
+
+/**
+ * A TimePicker component used to choose the time, following specifications provided by Design System. Still in development.
```

Comment:
Please remove the "Still in development".
Maybe the "following specifications provided by Design System" is redundant.
It seems good enough just "A TimePicker component used to choose the time."

Reply:
Well spotted, I'll remove it.